### DISCUSSION TITLE
Custom domain with vercel domain

### DISCUSSION THREAD

**Discussion by MejanH** [vercel/vercel#5035] [Help] [answered]
Hi,

After adding a custom domain, should i remove *.vercel.app domain from my project domain list?

should I keep it for better support? or is it going to affect SEO?

**Comment by styfle** [comment, marked as answer]
You can remove it if you don't want to use it.

You could also keep the vercel.app domain and redirect it your main domain.

https://vercel.com/docs/custom-domains#redirecting-domains
